blocks/quilted-image-block/features/quilted-image/default.jsx

```diff
@@ -0,0 +1,230 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { useContent } from "fusion:content";
+import { useFusionContext } from "fusion:context";
+
+// Arc Themes Components - Base set of components used to compose blocks
+// https://github.com/WPMedia/arc-themes-components/
+import {
+	Heading,
+	HeadingSection,
+	Image,
+	Button,
+	Paragraph,
+	Stack,
+} from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-quilted-image";
+function QuiltedImage({ customFields }) {
+	const {
+		headline,
+		image1URL,
+		overlay1Text,
+		overlay1TextVariant = "dark",
+		button1Text,
+		// NOTE : Link href properties currently need a default or "startsWith of undefined" error is raised.
+		// https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.00/src/components/link/index.jsx#L22
+		button1Action = "",
+		button1Variant = "dark",
+		image2URL,
+		overlay2Text,
+		overlay2TextVariant = "dark",
+		button2Text,
+		button2Action = "",
+		button2Variant = "dark",
+		image3URL,
+		overlay3Text,
+		overlay3TextVariant = "dark",
+		button3Text,
+		button3Action = "",
+		button3Variant = "dark",
+	} = customFields;
+
+	// Translate user-facing variant name into values component is expecting.
+	const getButtonVariant = (value) => {
+		return value === "dark" ? "primary" : "secondary";
+	};
+
+	// get properties from context for using translations in intl.json
+	// See document for more info https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2538275032/Lokalise+and+Theme+Blocks
+	// const { arcSite } = useFusionContext();
+	// const { locale } = getProperties(arcSite);
+	// const phrases = getTranslatedPhrases(locale);
+
+	// Account for "dark" or "light" variants y
+	/*
+  const cssClasses = [
+		BLOCK_CLASS_NAME,
+		variant === "dark" ? `${BLOCK_CLASS_NAME}--dark` : `${BLOCK_CLASS_NAME}--light`,
+	].join(" ");
+  */
+
+	return (
+		<div className={`${BLOCK_CLASS_NAME}`}>
+			<HeadingSection>
+				{headline ? <Heading>{headline}</Heading> : null}
+				<Stack>
+					<Stack direction="horizontal">
```

Comment:
This `<Stack>` has `direction="horizontal"` set temporarily as its initial, mobile-first state will be vertical. I understand that the direction change can/will be set in the feature pack SCSS with `default()` and `desktop()` maps. Should there be anything in the block's `_index.scss` (e.g. media queries) that will change the `<Stack>` direction as a default?
![image](https://user-images.githubusercontent.com/452134/160843718-64ea6e08-6cff-4424-b2c7-50e4e16d01a0.png)

Reply:
The Scss will be unable to change the direction of the stack based on viewport without doing a lot of custom overrides, we should also not be changing a prop via Scss which isn't possible

Reply:
I don't know that the stack direction should matter for the quilted item.  I think the only concern for quilted layout is the "primary" element position.  (primary first vs. primary last).  Not even sure that Stack is appropriate here since it's more of a grid than a stack.  With the Stack, wouldn't we also run into issues with mis-sized elements if the left image was bigger, smaller or a different aspect ratio?

Reply:
@matthewroach : Is the direction of a `<Stack>` not intended to be changed over breakpoints?  `<Stack>` is essentially a Flexbox container. Changing the flex-direction for layout purposes is a common CSS use case.

@nschubach : Agree that Grid is another valid possible approach. My concern would be making it the only option. Valid point on image sizes. I have the Stack for the first 2 images set to `flex: 1` to keep them the size width. I imagine that we'll have more reliance on image resizer v2 for this in future days.

Reply:
@ecarlisle it sounds like `<Stack> is essentially a Flexbox container. Changing the flex-direction of a Flexbox container for layout purposes is a common CSS use case.` is not intended based on @matthewroach 's comment